### DISCUSSION TITLE
Czech language - json file

### DISCUSSION THREAD

**Discussion by lukas-krivda** [Peppermint-Lab/peppermint#367] [Show and tell]
Hi, 

I have created a file with czech language. Maybe you can add new language to your project. Some things is not traslated.

[peppermint.json](https://github.com/user-attachments/files/16887141/peppermint.json)

![peppermint](https://github.com/user-attachments/assets/540809e1-8e3a-49ea-8954-b32eb444799e)

And some things are in english. Iťs possible change in code

Thank you for super project

L.

**Comment by potts99** [comment]
Thank you will add this!
